.github/workflows/dependabot.yml

```
@@ -0,0 +1,10 @@
+version: 2
+updates:
+  - package-ecosystem: "github-actions"
+    directory: "/"
+    schedule:
+      interval: "hourly"
```

```suggestion
      interval: "weekly"
```

I don't think [hourly is an option](https://docs.github.com/en/code-security/dependabot/dependabot-version-updates/configuration-options-for-the-dependabot.yml-file#scheduleinterval). 

How about weekly? In the past, I have found daily to be a bit overwhelming where dependabot makes a lot of PRs and it's annoying to keep up with.

---

Yes, it is going to be weekly, I set it to hourly so that we can test it